.github/workflows/dart-tests.yaml

```diff
@@ -298,3 +298,17 @@ jobs:
       - name: Run unit tests
         run: dart test --concurrency=1
         working-directory: ${{ matrix.path }}
+
+  integration_test_flutter:
+    name: Serverpod integration tests (flutter)
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        flutter-version: ['3.19.0', '3.24.0']
```

Comment:
Question: The matrix here mentions flutter version 3.19.0 and 3.24.0 but in the dockerfile we specifically install a single version of flutter, version 3.19, is the matrix really required for this test group?

Reply:
You are right, we dont need this, thanks!